driver/flambda_backend_args.ml

```diff
@@ -1335,6 +1348,12 @@ module Extra_params = struct
     match name with
     | "internal-assembler" -> set' Flambda_backend_flags.internal_assembler
     | "dgc-timings" -> set' Flambda_backend_flags.gc_timings
+    | "no-mach-ir" ->
+      Flambda_backend_flags.cfg_selection := true;
+      Flambda_backend_flags.cfg_cse_optimize := true;
+      Flambda_backend_flags.cfg_zero_alloc_checker := true;
+      Flambda_backend_flags.regalloc := "cfg";
```

Comment:
factor out into a function, just in case there are more flags to set in the future.

Reply:
The polling pass on CFG has no special flag because it's implied by the register allocator selection.